app/controllers/concerns/joiner.rb

```diff
@@ -60,11 +60,11 @@ def join_room(opts)
       opts[:mute_on_start] = room_setting_with_config("muteOnStart")
 
```

Comment:
Can you just set the avatar here? `opts[:avatarURL] = ....` 

That way you won't need to make any changes to `bbb_server.rb`

Reply:
> That way you won't need to make any changes to bbb_server.rb

No change at all? I thought we need to set join_opts[:avatarURL] anyway. Please look at my corrections and let me know if they make sense to you.